build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+set -v
```

Comment:
I would use `set -xe` just because this is what we usually use (I never saw `-v` before, it seems like the only difference is `-x` prepends also a `+` to each command).